src/client/index.js

```diff
@@ -0,0 +1,160 @@
+// based on https://github.com/ampproject/amp-wp/pull/1519
+
+/**
+ * Internal dependencies
+ */
+import { fetchDocument } from './fetch';
+import { hashDOMNode, compareDOMNodeCollections, fireEvent } from './utils';
+
+/**
+ * Attach event handlers to the document.
+ */
+function attachNavigationHandlers() {
+	document.body.addEventListener('click', handleClick);
```

Comment:
I wonder could this be why it's not working correctly when the image links from the homepage post block are clicked? Does the click event handler process the`img` element and not the `a` element when images wrapped in a link are clicked? I can do further testing around it tomorrow if you can't reproduce the error.

Reply:
(answered in a general comment)